server/controllers/webhooks.js

```diff
@@ -37,6 +37,9 @@ module.exports = (app) => {
 
     // Stripe send test events to production as well
     // don't do anything if the event is not livemode
+    if (!body.hasOwnProperty("livemode")) {
+      console.error("Assertion error: expected a body.livemode property!");
+    }
```

Comment:
Why don't add this `console.error` within the `if (isProduction && !body.livemode)`?